issue_06/script.js

```diff
@@ -11,21 +11,27 @@ const contents = [{
 }];
 
 const ul = document.getElementById('js-ul');
-const checkContents = new Promise((resolve) => {
-    resolve(contents);
-});
-checkContents.then((value) => {
+const wait = (sec) => {
+    return new Promise((resolve) => {
+        setTimeout(resolve, sec * 1000);
+    });
+}
+
+async function createLists() {
+    await wait(3);
```

Comment:
You can wait 3 seconds without writing this part.

```
  setTimeout(() => {
    resolve(contents);
  }, 3000);
})
```

Reply:
I couldn't get rid of the await description for the reason of waiting for the process in async.
However, I felt that setting extra arguments would make the code more difficult to read, so I changed the description as you suggested.
https://github.com/yoshikawa-akr/js-lessons/pull/6/commits/5e2cb0adcc2df2601b0b852975d3232d81cd26c5

Reply:
nits

> createList.then((value) => {
  for (const item of value) 

If you write it like this, you will not need the async function.

Reply:
Actually you should write this task without async await function. Because the async await function is in a much further issue. The challenge here is to get familiar with the Promise object.

Reply:
Thanks for the comment!
I have corrected the points that you both pointed out.
https://github.com/yoshikawa-akr/js-lessons/pull/6/commits/92044e8aa6aa6ae04895e4e4910a32161145c3eb